config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
+
+en:
```

Comment:
question: Do you understand why we might need locales and a separate file for each locale?

Reply:
From what I understood, rails and devise support translations to different languages. So I suppose different files is to define things in different languages.